tests/test_daemon.py

```
@@ -1,300 +1,135 @@
+
+
+def test_telemetry_skipped_when_sent_recently(tmp_path: Path) -> None:
+    """Telemetry should skip if last_sent is within 24h."""
+    from datetime import datetime, timezone
+
+    fake_home = tmp_path / "fakehome"
+    config_dir = fake_home / ".gradata"
+    config_dir.mkdir(parents=True)
+    config_path = config_dir / "config.toml"
+
+    now = datetime.now(timezone.utc).isoformat()
```

Ruff (0.15.9)

[warning] Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)